components/observations/ObservationDetailView.tsx

```diff
@@ -309,6 +306,7 @@ export const ObservationCard: React.FunctionComponent<{
                       </Marker>
                     </ZoneMap>
                   )}
+                  <TableRow label="Avalanche Center" value={observation.center_id} />
```

Comment:
I suggest we add this into the Obs page, regardless of logo in title/etc. I think this makes it easier to know what center the obs is shared from, in case you dont know what other center logos look like/etc since we do take you back to your default center. I am going to see what Charlotte thoughts are on this.

Reply:
nit: we need the display name with `userFacingCenterId`